benchflow-dsl/src/main/scala/cloud/benchflow/dsl/dockercompose/DockerComposeYamlBuilder.scala

```diff
@@ -0,0 +1,67 @@
+package cloud.benchflow.dsl.dockercompose
+
+import cloud.benchflow.dsl.definition.types.bytes.Bytes
+import cloud.benchflow.dsl.definition.types.bytes.BytesYamlProtocol._
+import net.jcazevedo.moultingyaml.{ YamlString, YamlValue, _ }
+
+/**
+ * @author Jesper Findahl (jesper.findahl@gmail.com)
+ *         created on 2017-05-25
+ */
+class DockerComposeYamlBuilder(dockerComposeYamlValue: YamlValue) {
+
+  val ServicesKey = YamlString("services")
+  val MemLimitKey = YamlString("mem_limit")
```

Comment:
Is this the hard or the soft limit? Is it ok to support only the hard limit, but we should name the variables in a consistent way so that then we can extend the DSL.